examples/WolfSheep/WolfSheep.py

```diff
@@ -86,11 +92,41 @@ def __init__(self, height=20, width=20,
             self.grid.place_agent(wolf, (x, y))
             self.schedule.add(wolf)
 
+        # Create grass patches
+        if self.grass:
+            for agent, x, y in self.grid.coord_iter():
+
+                fully_grown = random.choice([True, False])
+
+                if fully_grown:
+                    countdown = self.grass_regrowth_time
+                else:
+                    countdown = random.randrange(self.grass_regrowth_time)
+
+                patch = GrassPatch(fully_grown, countdown)
+                self.grid.place_agent(patch, (x, y))
+                self.schedule.add(patch)
+
         self.running = True
 
     def step(self):
         self.schedule.step()
+        print([self.schedule.time,
```

Comment:
Printing should be optional (e.g. with a `verbose` parameter) since otherwise it clutters up the command line during batch runs, visualization, etc. This is fine for debugging, but best practice is probably to use a data collector to keep track of these stats.